news/2021-09-18-200th-featured-artist-our-stolen-theory.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "200th Featured Artist: Our Stolen Theory"
+date: 2021-09-18 09:30:00 +0000
+---
+
+Celebrating a massive milestone with the official inclusion of ***Our Stolen Theory** - United (L.A.O.S Remix)* in the Featured Artist program!
```

Comment:
this sentence feels very weird imo, i think you meant to write sth like this here

`We're celebrating a massive milestone with the official inclusion of ***Our Stolen Theory** - United (L.A.O.S Remix)* in the Featured Artist program!`

Reply:
wow we had a wavelength moment, fixed this while you were making this comment